### ISSUE TITLE
Support for subresource integrity

### ISSUE THREAD

**Issue by httpsOmkar** [open-wc/open-wc#1502]
As per security consideration, We can use the sub resource integrity which can improve the security of app.

**Comment by LarsDenBakker** [comment]
Could you give some note details?

**Comment by httpsOmkar** [comment]
If we implement this https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity into the rollup build.
Means we create hash of files and put into the integrity attributes like
```html
<script src="https://example.com/example-framework.js"
        integrity="sha384-oqVuAfXRKap7fdgcCY5uykM6+R9GqQ8K/uxy9rx7HNQlGYl1kPzQho1wx4JwY8wC"
        crossorigin="anonymous"></script>
```
If some attacker change the content of the js the browser will stop to load the content.
Which will in-result improve the security of the application.
For webpack I was using this plugin which automatically this hash into the index.html
https://www.npmjs.com/package/webpack-subresource-integrity

**Comment by LarsDenBakker** [comment]
I agree we should do something here, need to dig into this topic to understand the requirements and possibilities.

We just released a new version of the rollup config, including a new HTML plugin which allows customising how the final script is injected into the page. So you should be able to implement this yourself for now.
